contracts/LvlV1.sol

```diff
@@ -18,8 +18,8 @@ contract Level is ERC721, Skills, Multicall, Ownable {
   // Mapping of token IDs by owner address:
   mapping(address => uint256) private _tokenByOwner;
 
-  constructor() ERC721("Level", "LEVEL") {
-    _baseURIExtended = "https://level.2c.io/api/token/";
+  constructor() ERC721("Lvl", "LVL") {
+    _baseURIExtended = "https://lvl.2c.io/api/token/";
```

Comment:
should this point to lvlprotocol.xyz instead?

Reply:
Good call @edderrd !

Reply:
We can change this with the setBaseURI, so we should be good. Good call though!!